tests/php/Jobs/CloneImagesOrVideosTest.php

```diff
@@ -29,30 +29,31 @@ class CloneImagesOrVideosTest extends \ApiTestCase
     public function testCloneImageVolume()
     {
         Event::fake();
-        $volume = $this
-          ->volume(
-            ['created_at' => '2022-11-09 14:37:00',
+        $volume = 
+            $this->volume(
+                ['created_at' => '2022-11-09 14:37:00',
                 'updated_at' => '2022-11-09 14:37:00',])
-          ->fresh(); // Use fresh() to load even the null fields.
+            ->fresh(); // Use fresh() to load even the null fields.
 
         $copy = $volume->replicate();
         $copy->save();
         // The target project.
         $project = ProjectTest::create();
         $project->addVolumeId($copy->id);
 
-        $oldImage = ImageTest::create([
-            'filename' => 'a123.jpg',
-            'taken_at' => Carbon::now()->setTimezone('Europe/Lisbon'),
-            'volume_id' => $volume->id,
-            'lng' => 1.5,
-            'lat' => 5.3,
-            'tiled' => true
+        $oldImage = 
+            ImageTest::create([
+                'filename' => 'a123.jpg',
+                'taken_at' => Carbon::now()->setTimezone('Europe/Lisbon'),
+                'volume_id' => $volume->id,
+                'lng' => 1.5,
+                'lat' => 5.3,
+                'tiled' => true,
             ])
             ->fresh();
-        ImageLabelTest::create(['image_id' => $oldImage->id]);
+        ImageLabelTest::create(['image_id' => $oldImage->id,]);
```

Comment:
I only meant for you to add trailing commas to the positions I indicated, not all arrays in this file. Sorry if that was confusingly phrased. Please remove all training commas of single-line arrays again (i.e. commas that are directly followed by a `]`).